packages/wdio-reporter/src/index.js

```diff
@@ -183,11 +184,68 @@ export default class WDIOReporter extends EventEmitter {
         return true
     }
 
+    /**
+     * Returns everything worth reporting from a suite
+     * @param  {Object}    suite  test suite containing tests and hooks
+     * @return {Object[]}         list of events to report
+     */
+    getEventsToReport (suite) {
+        return [
+            /**
+             * report all tests
+             */
+            ...suite.tests,
+            /**
+             * and only hooks that failed
+             */
+            ...suite.hooks
+                .filter((hook) => Boolean(hook.error))
+        ]
+    }
+
+    /**
+     * Get display for failed tests, e.g. stack trace
+     * @return {Array} Stack trace output
+     */
+    getFailureDisplay () {
+        let failureLength = 0
+        const output = []
+
+        for (const [ , suite ] of Object.entries(this.all_suites)) {
+            const suiteTitle = suite.title
+            const eventsToReport = this.getEventsToReport(suite)
+            for (const test of eventsToReport) {
+                if(test.state !== 'failed') {
+                    continue
+                }
+
+                const testTitle = test.title
+
+                // If we get here then there is a failed test
+                output.push(
+                    '',
+                    `${++failureLength}) ${suiteTitle} ${testTitle}`,
+                    chalk.red(test.error.message),
+                    ...test.error.stack.split(/\n/g).map(value => chalk.gray(value))
```

Comment:
Can we please pull the update from wdio-spec reporter which handles multiple errors if they are present on the test? Jasmine will emit more than one error per test in an `errors`. See [https://github.com/webdriverio/webdriverio/pull/3702/files](https://github.com/webdriverio/webdriverio/pull/3702/files)